Resources/Prototypes/_Impstation/Entities/Objects/Weapons/Guns/Ammunition/Boxes/Lpistol.yml

```diff
@@ -0,0 +1,85 @@
+- type: entity
+  abstract: true
+  parent: BaseItem
```

Comment:
same deal as the lpistol comment. could make this parent a different ammo container. etc etc etc. the parent system kind of fucks crazy once you learn it